Makefile

```diff
@@ -0,0 +1,51 @@
+GO ?= $(shell command -v go 2> /dev/null)
+MATTERMOST_CLOUD_IMAGE ?= mattermost/mattermost-cloud:test
+MACHINE = $(shell uname -m)
+BUILD_IMAGE = golang:1.12
+BASE_IMAGE = alpine:3.9
+GOPATH ?= $(shell go env GOPATH)
+GOFLAGS ?= $(GOFLAGS:)
+BUILD_TIME := $(shell date -u +%Y%m%d.%H%M%S)
+BUILD_HASH := $(shell git rev-parse HEAD)
+
+export GO111MODULE=on
+
+## Checks the code style, tests, builds and bundles.
+all: check-style dist
+
+## Runs govet and gofmt against all packages.
+.PHONY: check-style
+check-style: lint govet
```

Comment:
I think `govet` should be run first, since it will complain about compilation errors that `lint` might choke on too.